hles/extraction/src/main/scala/org/broadinstitute/monster/dap/OkWrapper.scala

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.monster.dap
+
+import java.io.IOException
+import java.time.Duration
+
+import okhttp3.{Call, Callback, OkHttpClient, Request, Response}
+import org.broadinstitute.monster.common.msg.JsonParser
+import upack.Msg
+
+import scala.concurrent.{Future, Promise}
+
+class OkWrapper extends HttpWrapper {
+  /** Timeout to use for all requests to production RedCap. */
+  private val timeout = Duration.ofSeconds(60)
+
+  /** Construct a client instance backed by the production RedCap instance. */
+  val client = new OkHttpClient.Builder()
```

Comment:
This causes a serialization error when running the pipeline via dataflow. If you make wrap it in a  `def` or `lazy val` that'll defer instantiation until first access in `makeRequest`